src/test/codewhisperer/util/editorContext.test.ts

```diff
@@ -7,11 +7,21 @@ import assert from 'assert'
 import * as codewhispererClient from '../../../codewhisperer/client/codewhisperer'
 import * as EditorContext from '../../../codewhisperer/util/editorContext'
 import { createMockTextEditor, createMockClientRequest, resetCodeWhispererGlobalVariables } from '../testUtil'
+import globals from '../../../shared/extensionGlobals'
+import { GenerateCompletionsRequest } from '../../../codewhisperer/client/codewhispereruserclient'
 
 describe('editorContext', function () {
+    let telemetryEnabledDefault: boolean
+
     beforeEach(function () {
         resetCodeWhispererGlobalVariables()
+        telemetryEnabledDefault = globals.telemetry.telemetryEnabled
+    })
+
+    afterEach(function () {
+        globals.telemetry.telemetryEnabled = telemetryEnabledDefault
```

Comment:
don't quite understand this change, does this will really change the value of `globals.telemetry.telemetryEnabled` because I dont' see `telemetryEnabledDefault` is assigned a value elsewhere

Reply:
The tests need to change `globals.telemetry.telemetryEnabled`, the beforeEach and afterEach stores and resets this value before and after every time a test execution.